docs/mat4.js.html

```diff
@@ -974,6 +974,38 @@ <h1 class="page-title">Source: mat4.js</h1>
   return out;
 }
 
+//TODO FIX ME
```

Comment:
Is this outstanding?

Reply:
That's a documentation file. I didn't rebuild the docs. (Should probably do it)